crypto/keys/internal/ecdsa/init.go

```diff
@@ -0,0 +1,3 @@
+// Package ECDSA implements Cosmos-SDK compatible ECDSA public and private key. The keys
+// can be serialized.
+package ecdsa
```

Comment:
Why a separate file?

Reply:
doc.go?

Reply:
before there was more stuff in this file, so it left alone. I can rename to `doc.go`, unless you have a preference to move the doc comment to pubkey.go

Reply:
renamed to doc.go